constants.js

```diff
@@ -19,5 +19,11 @@ export const Bond = memoize((nation, number) => ({
   number,
   cost: cost[number],
 }));
+export const AllBonds = new Set();
+["AH", "IT", "FR", "GB", "GE", "RU"].forEach((nation) =>
+  Object.keys(cost).map((number) =>
+    AllBonds.add(Bond(Nation[nation], parseInt(number)))
+  )
+);
```

Comment:
It seems every time we use `AllBonds`, we are cloning it via `[...AllBonds]`. Why not turn this into a function?
```suggestion
export const AllBonds = () => new Set(
["AH", "IT", "FR", "GB", "GE", "RU"].map((nation) =>
  Object.keys(cost).map((number) =>
    Bond(Nation[nation], parseInt(number))
  )
).flat());
```